modules/custom/d_commerce_products_list/d_commerce_product_list.install

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @file
+ * Install, update, and uninstall functions for d_commerce_product_list.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function d_commerce_product_list_install() {
+  $theme_name = \Drupal::config('system.theme')->get('default');
```

Comment:
To przy okazji `\Drupal::config` to to samo co` \Drupal::service('config.factory')->get;` wiec skoro ponizej juz ładujesz config service, to moze lepiej pierw załadować serwis, a potem załadować sobie przez niego także config `system.theme`?